Automattic/wp-calypso
# Invites: new action to create and send invites

Setting up an action / api call that will allow an admin to invite people to contribute to their blog.

To test:
- check out the branch
- select a wpcom site, and click 'Add' next to people in the sidebar
- you should see a temporary UI that lets you try out the new API endpoint
- try sending some invites to your site
- note the response, it should report that the invites were sent or not sent
- sent invites should be in your inbox!

cc: @ebinnion @lezama
